Installer/51-gcadapter.rules

```diff
@@ -0,0 +1 @@
+SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ATTRS{idVendor}=="057e", ATTRS{idProduct}=="0337", MODE="0666"
```

Comment:
That's not what the games group is for. 

It is possible to give permission to only the people logged on at a physical seat, but the setup for that is per-distro. 0666 is the best cross-distro method.